assets/airports/kbos.json

```diff
@@ -63,12 +65,12 @@
         "_LOGAN22R":  ["N42.30994664062", "W70.92237384752"],
```

Comment:
Up above are the `_15R`, `_22L`, `_22R`, `_27`, and `_33L` fixes... I'd like them to be changed to conform with a more standard and universally obvious convention:

  - The fixes should be located at the thresholds of the runways for which they are named.
    - Example: `"_RWY33L": [42.354662, -70.991598]`
  - Any fixes desired a given distance away from another fix will be described in fix-radial-distance form
    - this would be the fix name, three digit bearing, and three digit distance in nautical miles
    - all of these should be marked as RNAV fixes (via the underscore prefix)
    - Example: `"_AUTUM220015": [42.324333, -71.736833]`
  - Any fixes desired a given distance out on final of a given runway will be described via the distance from the threshold
    - this would be the ruinway name, two digit distance in nautical miles, then `DME`
    - all of these should be marked as RNAV fixes (via the underscore prefix)
    - Example: `"_RWY33L01DME": [42.342838, -70.975751]`